### PR TITLE
area updates

### PR DESCRIPTION
**Pull request by stonedDiscord**
did i forget anything

### CODE REVIEW

File: webAO/client.js [hunk 1]
```diff
@@ -879,11 +887,61 @@ class Client extends EventEmitter {
...
+						case "GAMING":
+							thisarea.classList = "area-button area-gaming";
+							break;
+						case "SPECTATABLE":
+							thisarea.classList = "area-button area-spectatable";
+							break;
+						default:
+							thisarea.classList = "area-button area-default";
+							break;
+					}
```
Comment (oldmud0) [inline webAO/client.js]: coulda used a table bruh - always consider table lookup over switch/case

File: webAO/client.js [hunk 2]
```diff
@@ -879,11 +887,61 @@ class Client extends EventEmitter {
...
+
+				}
+				break;
+			case "2":
+				break;
+			case "3":
+				break;
```
Comment (oldmud0) [inline webAO/client.js]: I forgot what these are - cm change?

Comment (stonedDiscord) [reply webAO/client.js]: yes it's cm

**Comment by stonedDiscord** [comment]
better?
i will merge this if no one _objects_

**Pull request merged**